### ISSUE TITLE
Duplicate install_requires in setup.py

### ISSUE THREAD

**Issue by jhermann** [getsentry/sentry#4934]
Is there some hidden logic behind duplicating all requirements in the ``postgres`` extras? This makes e.g. the list on the PyPI page even longer than it already is, and also somewhat confusing. ;)

```py
    install_requires=install_requires,
    extras_require={
        'dev': dev_requires,
        'postgres': install_requires,
        'tests': tests_require,
    },
```

**Comment by DoronCohen** [comment]
Hi! I want to contribute and this is my first PR. What should be done here? just remove the line and test? should it remain for compat but get an empty list?

**Comment by jhermann** [comment]
'postgres': [],

As long as there is no other DB option, pgsql stuff can stay in the main list. And as you noticed, the extra name must stay for not breaking stuff out there.

**Comment by mattrobenolt** [comment]
Yeah, this is literally just a one-line change. @DoronCohen if you wanna submit a PR for this, that's great, otherwise, I can just do it.

**Comment by DoronCohen** [comment]
I will submit one. Thanks

**Comment by mattrobenolt** [comment]
Fixed by #5057
